fastn-core/Cargo.toml

```diff
@@ -80,6 +80,8 @@ fastn-observer.workspace = true
 fastn-js.workspace = true
 argon2.workspace = true
 uuid.workspace = true
+lettre.workspace = true
```

Comment:
I think this is a decent idea as well. We will soon have a sort of "fastn standard environment", which will provide PostgreSQL, Redis, and SMTP.